### DISCUSSION TITLE
Case sensitive search at cs.github.com

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#9825] [Other Feature Feedback, Questions, & Ideas]
It seems that the search is case insensitive:

Searching:
```
".terminal(" language:javascript
```
it finds:

apache/zeppelin with code:

```javascript
new hterm.Terminal();
```

Even regex that is (in most languages) case sensitive doesn't work:

```
/\.terminal\(/ language:javascript
```

it still finds Zeppelin

It would be nice to have a way to make it case-sensitive.

**Comment by Drarig29** [comment]
I agree, and it would be even cooler to be able to use a case sensitive (the default) regex, and if we want, a `/regex/i` to have a case insensitive one.

**Comment by shric** [comment]
Duplicate of https://github.com/github-community/community/discussions/9759
